packages/notifications/src/Provider.spec.tsx

```diff
@@ -1,6 +1,8 @@
 import { act, renderHook, render, screen, userEvent } from '@ttoss/test-utils';
 
-import { useNotifications } from './';
+import { useNotifications, TOAST_TYPES } from '.';
+
+jest.useFakeTimers();
```

Comment:
Just for you know, you can set `timer: fake` on `jest.config.ts` instead of using `jest.useFakeTimers()`.

https://jestjs.io/docs/configuration#timers-string